internal/pkg/template/templates/workloads/services/worker/manifest.yml

```diff
@@ -42,6 +42,10 @@ subscribe:
     - name: {{$topic.Name}}
       service: {{$topic.Service}}
 {{- end}}
+  {{- if and .Subscribe.Queue .Subscribe.Queue.FIFO .Subscribe.Queue.FIFO.Enable }}
```

Comment:
We might just need
```suggestion
  {{- if .Subscribe.Queue.FIFO.Enable }}
```

Reply:
I think we need this because there is no check before this point for the `Queue` and `FIFO` configs. If they turned out to be nil then we will face nil pointer exception.

Reply:
Can they turn `nil` 🤔

Reply:
But in CFN we use struct from `template/workflow` like [this](https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/template/workload.go#L435). So it can be nil.

Reply:
im not convinced. Your url is for cfn template generation and here we are talking about manifest, which is marshaling the manifest struct.

Reply:
Ohh you are right, I got confused with the templates. Thanks for catching this.